client/state/sites/test/selectors/get-site-options.js

```diff
@@ -1,13 +1,21 @@
-/**
- * External dependencies
- */
-import { expect } from 'chai';
-
 /**
  * Internal dependencies
  */
-import getSiteOptions from 'calypso/state/selectors/get-site-options';
-import { userState } from './fixtures/user-state';
+import { getSiteOptions } from '../..';
```

Comment:
Should we replace `../../` with `calypso/state/selectors/sites` to be more explicit?